start.sh

```diff
@@ -1,8 +1,17 @@
 #!/bin/bash
-for d in */ ; do
-    if [ "$d" != "node_modules/" ]
-    then
-        echo "${d}"
-        testcafe chrome "${d}index.js"
-    fi
-done
\ No newline at end of file
+if [ $# -gt 0 ]
+then
+  for dir in "$@"; do
```

Comment:
co to za czary linuxowo - bashowe XDD
ale i tak bym dodał, żeby ktoś sobie w opcjach nie wpisał `node_modules` XDDD

Reply:
okej XD